src/constants/errors.ts

```diff
@@ -0,0 +1,6 @@
+import { getBytesAsString } from "utils/string";
+import { LOG_LINE_SIZE_LIMIT } from "./logs";
+
+export const LOG_LINE_TOO_LARGE_WARNING = `Parsley was unable to process one or more lines due to performance reasons since they exceed the line size limit of ${getBytesAsString(
```

Comment:
Performance reasons feels implied, may as well keep the toast concise
```suggestion
export const LOG_LINE_TOO_LARGE_WARNING = `Parsley was unable to process one or more lines since they exceed the line size limit of ${getBytesAsString(
```